syncmaster/worker/transfer.py

```diff
@@ -4,20 +4,25 @@
 from datetime import datetime, timezone
 
 import onetl
+from asgi_correlation_id import correlation_id
+from celery.signals import before_task_publish, task_prerun
 from sqlalchemy import select
 from sqlalchemy.orm import Session, selectinload
 
-from syncmaster.config import Settings
 from syncmaster.db.models import AuthData, Run, Status, Transfer
 from syncmaster.db.repositories.utils import decrypt_auth_data
 from syncmaster.exceptions.run import RunNotFoundError
+from syncmaster.settings import Settings
 from syncmaster.worker.base import WorkerTask
 from syncmaster.worker.config import celery
 from syncmaster.worker.controller import TransferController
 
 logger = logging.getLogger(__name__)
 
 
+CORRELATION_CELERY_HEADER_ID = Settings().CORRELATION_CELERY_HEADER_ID
```

Comment:
Please never instantiate Settings() during the import. This should be done somewhere in `__main__` function.
If it's hard to implement this for now, add TODO

Reply:
added TODO